metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1087,10 +1106,60 @@ class MetalsLanguageServer(
             )
           )
         }.asJavaObject
+      case ServerCommands.StartDebugAdapter() =>
```

Comment:
Any chance to move it to an outside provider? There is already a lot of code inside the server, let's try not to add more.